recipes/libcf/build.sh

```diff
@@ -0,0 +1,23 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC ${CFLAGS}"
+export CXXLAGS="${CFLAGS} ${CXXLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export LFLAGS="-fPIC ${LFLAGS}"
+
+if [ "$(uname)" == "Darwin" ]; then
+    export CXXFLAGS="${CXXFLAGS} -fno-common"
+    export MACOSX_DEPLOYMENT_TARGET=$(sw_vers -productVersion | sed -E "s/([0-9]+\.[0-9]+).*/\1/")
```

Comment:
I hope your activate/deactivate works.  This is a hard one to compile on OSX.
[toolchain](https://github.com/conda-forge/toolchain-feedstock/tree/master/recipe)